includes/documents/class-wcpdf-invoice.php

```diff
@@ -103,27 +103,25 @@ public function init_number() {
 			return $invoice_number;
 		}
 
-		$number_store_method = WPO_WCPDF()->settings->get_sequential_number_store_method();
-		$number_store_name   = $this->store_name();
-		$number_store        = new Sequential_Number_Store( $number_store_name, $number_store_method );
-		$invoice_number      = $number_store->increment( $this->order_id, $this->get_date()->date_i18n( 'Y-m-d H:i:s' ) );
+		$store_name     = $this->get_store_name();
+		$number_store   = $this->get_sequential_number_store( $store_name );
+		$invoice_number = $number_store->increment( $this->order_id, $this->get_date()->date_i18n( 'Y-m-d H:i:s' ) );
 
 		$this->set_number( $invoice_number );
 
 		return $invoice_number;
 	}
 
-	public function store_name() {
+	public function get_store_name() {
 		$document_date       = $this->get_date()->date_i18n( 'Y' );
 		$current_date        = date( 'Y' );
 		$reset_numeration    = isset( $this->settings['reset_number_yearly'] ) ? true : false;
 		$current_store_name  = "{$this->slug}_number_{$document_date}";
-		$number_store_method = WPO_WCPDF()->settings->get_sequential_number_store_method();
 
 		// reset: on
 		if( $reset_numeration ) {
 			$store_name   = $current_store_name;
-			$number_store = new Sequential_Number_Store( $store_name, $number_store_method );
+			$number_store = $this->get_sequential_number_store( $store_name );
```

Comment:
this makes more sense the other way around, `get_sequential_number_store()`  calling `get_store_name()`

Reply:
That is, if you need the separation at all!